rotkehlchen/api/v1/resources.py

```diff
@@ -153,7 +153,13 @@
     XpubAddSchema,
     XpubPatchSchema,
 )
-from rotkehlchen.assets.asset import Asset, AssetWithNameAndType, AssetWithOracles, CustomAsset
+from rotkehlchen.assets.asset import (
+    Asset,
+    AssetWithNameAndType,
+    AssetWithOracles,
+    CustomAsset,
+    ResolvedAsset,
+)
```

Comment:
nitpick: all this can be moved under if TYPE_CHECKING since they are just used for typing